### ISSUE TITLE
v0.15.0

### ISSUE THREAD

**Issue by SergeyMiracle** [apicase/adapter-fetch#5]
Having this error `TypeError: res.headers.entries(...).reduce is not a function`
It happens here 
```
const createResponse = res => body => ({
  status: res.status,
  headers: res.headers.entries().reduce((res, pair) => {
    res[pair[0]] = pair[1]
    return res
  }, {}),
  body: body
})
```

**Comment by Kelin2025** [comment]
Woops, sorry
Fixed in `0.15.1`

**Comment by SergeyMiracle** [comment]
Thx 😃
